alphametics.json

```diff
@@ -21,6 +21,16 @@
                 "description": "solve puzzle with exponents",
                 "puzzle": "PI * R ^ 2 == AREA",
                 "expected": { "P": 9, "I": 6, "R": 7, "A": 4, "E": 0 }
+            },
+            {
+                "description": "invalid solution returns null",
```

Comment:
Good catch, the reason that this title was 'returns null' was to set up for the 'leading zero' language, the explanation as to why it returns null, is due to being invalid. 

Should he title be: "invalid solution of non-unique letter/value pair returns null"?  It is much more limiting, implying that is the reason for the null, rather than general "invalidness"?